packages/docs/src/pages/startup/installation.md

```diff
@@ -3,7 +3,7 @@ title: Install with NPM
 weight: 0
 ---
 
-The design system is available as NPM packages or via a <a href="https://github.com/CMSgov/design-system/archive/v1.30.0.zip">.zip download</a>.
```

Comment:
Old page directly linked to a release version. I updated to our normal release page